cmd/collector/app/span_handler_builder.go

```diff
@@ -51,6 +51,7 @@ func (b *SpanHandlerBuilder) BuildSpanProcessor(additional ...ProcessSpan) proce
 		Options.NumWorkers(b.CollectorOpts.NumWorkers),
 		Options.QueueSize(b.CollectorOpts.QueueSize),
 		Options.CollectorTags(b.CollectorOpts.CollectorTags),
+		//nolint: gosec // G115
 		Options.DynQueueSizeWarmup(uint(b.CollectorOpts.QueueSize)), // same as queue size for now
```

Comment:
we should change CollectorOpts.QueueSize to uint instead (negative queue size is invalid)